modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -82,4 +85,4 @@ Click the *Resource Limit* link to set the amount of *CPU* and *Memory* resource
 Labels::
 Click the *Labels* link to add custom labels to your application.
 
-. Click *Create* to create the application and see its build status in the *Topology* view.
+. Click *Create* to create the application and a toast notification is displayed. You can see its build status in the *Topology* view.
```

Comment:
'a toast notification is displayed' - the description of the type of notification is a background UI detail that does not add much value to the user. What would make sense to the user is something like, 'a success notification is displayed'.
Now that it is a separate sentence, suggest: You can see its build status.../You can see the build status of the application...